analyzer/src/main/kotlin/managers/Bower.kt

```diff
@@ -243,9 +243,11 @@ class Bower(
                 scopes = sortedSetOf(dependenciesScope, devDependenciesScope)
             )
 
-            return ProjectAnalyzerResult(
-                project = project,
-                packages = packages.mapTo(sortedSetOf()) { it.value.toCuratedPackage() }
+            return listOf(
+                ProjectAnalyzerResult(
+                    project = project,
+                    packages = packages.mapTo(sortedSetOf()) { it.value.toCuratedPackage() }
+                )
```

Comment:
For the reviewer's pleasure, it would have been slightly nicer to write this as

    val result = ProjectAnalyzerResult(
        // ...
    )

    return listOf(result)

as then the indentation of `ProjectAnalyzerResult` does not change which keeps the diff small. Same for the other package manager implementations. Would you mind making that change?

Reply:
I think the question is more which end result do you prefer? The way it is now, or the way you proposed above? IMO it's still not a complex diff. Keep in mind that you asked me to make the diff more complex in Pub.kt in another comment, I find that a bit contradictory.

Reply:
> I think the question is more which end result do you prefer?

I would have preferred the style I'm proposing, i.e. that's how I would have done it. But it's not a big enough issue on its own and I wouldn't have mentioned it if I didn't already have other comments.

> Keep in mind that you asked me to make the diff more complex in Pub.kt in another comment, I find that a bit contradictory.

To me the question that counts is: How easily can you simplify the diff (i.e. hide unimportant changes like indentation changes) without sacrificing functionality or elegance? Here, the diff can be trivially simplified without any impact on functionality or elegance. I `Pub.kt` the code you were touching already was not very elegant before, but it was easy to fix it along the way.

I probably should have written "hide unimportant changes" instead of "simplify the diff" to begin with to make this clearer. And to some extend I blame GitHub for not highlighting unimportant changes in its diffs.

Reply:
Ok, that makes sense to me, but since you approved I'll merge it as is.

Reply:
Yes, we could anyway change this later when a package manager starts to return multiple projects per definition file.